server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -557,6 +560,35 @@ private int assignedOrHosted(Map<TableId,TableCounts> last) {
     return result;
   }
 
+  // This method is used to detect if a tablet needs to be split for a delete
+  // Instead of performing a split, the tablet will have it's files fenced.
+  private boolean needsFencingForSplit(MergeInfo info, KeyExtent extent) {
+    // Merges don't split
+    if (!info.isDelete()) {
+      return false;
+    }
+    // Does this extent cover the end points of the delete?
+    KeyExtent range = info.getExtent();
+    if (extent.overlaps(range)) {
+      for (Text splitPoint : new Text[] {range.prevEndRow(), range.endRow()}) {
+        if (splitPoint == null) {
+          continue;
+        }
+        if (!extent.contains(splitPoint)) {
+          continue;
+        }
+        if (splitPoint.equals(extent.endRow())) {
+          continue;
+        }
+        if (splitPoint.equals(extent.prevEndRow())) {
+          continue;
+        }
+        return true;
+      }
+    }
+    return false;
+  }
+
```

Comment:
This method appears to only be used in one place, and can be consolidated into that method. I do that in my other comment, so this method can just be deleted.

```suggestion
```